src/audio/SDL_audiocvt.c

```diff
@@ -68,6 +72,7 @@ static int GetHistoryBufferSampleFrames(const int required_resampler_frames)
 {
     SDL_assert(required_resampler_frames <= RESAMPLER_MAX_PADDING_FRAMES);
 
+    // Even if we aren't currently resampling, make sure to keep enough history in case we need to later.
```

Comment:
```suggestion
    /* Even if we aren't currently resampling, make sure to keep enough history in case we need to later. */
```

Reply:
With SDL3 we can now use `//` style comments.

Reply:
Oh, forgot about that. Sorry about the noise.